reference/array/functions/array-diff.xml

```diff
@@ -12,13 +12,12 @@
   &reftitle.description;
   <methodsynopsis>
    <type>array</type><methodname>array_diff</methodname>
-   <methodparam><type>array</type><parameter>tablica1</parameter></methodparam>
-   <methodparam><type>array</type><parameter>tablica2</parameter></methodparam>
-   <methodparam choice="opt"><type>array</type><parameter>...</parameter></methodparam>
+   <methodparam><type>array</type><parameter>array</parameter></methodparam>
+   <methodparam rep="repeat"><type>array</type><parameter>arrays</parameter></methodparam>
   </methodsynopsis>
   <para>
-   Porównuje <parameter>tablica1</parameter> z jedną lub większą ilością tablic
-   i zwraca wartości <parameter>tablica1</parameter>, które nie są obecne w
+   Porównuje <parameter>array</parameter> z jedną lub większą ilością tablic
```

Comment:
```suggestion
   Porównuje tablicę <parameter>array</parameter> z jedną lub większą ilością tablic
```

chodzi mi o to, aby chociaż raz na stronę zaznaczyć że "array" to własnie tablica. Niby oczywiste, ale wydaje mi się że kiedy już tłumaczymy manual to warto o to zadbać. Dokłada jeden wyraz i jeśli zna się angielski, to wychodzi trochę masło maślane, ale unikamy niejasności, w wypadku gdyby ktoś jednak tego nie wiedział.

Analogiczna uwaga tyczy się też innych plików w tym PR, tych poniżej array_diff, bo wcześniejsze już poprawiliśmy,. Spróbuj po prostu przejrzeć pod tym kątem.